src/ui/inspector/InspectorViews.vue

```diff
@@ -33,9 +33,11 @@ export default {
 
             this.selectedViews = this.openmct.inspectorViews.get(selection);
             this.selectedViews.forEach(selectedView => {
-                let viewContainer = document.createElement('div');
-                this.$el.append(viewContainer)
-                selectedView.show(viewContainer);
+                if (typeof selectedView.tabbed === 'function' && !selectedView.tabbed()) {
```

Comment:
See previous comment, let's just assume we have two tabs for right now.

Reply:
Done.